cdap-ui/app/cdap/components/PipelineScheduler/Store/index.js

```diff
@@ -96,6 +107,20 @@ const schedule = (state = DEFAULT_SCHEDULE_OPTIONS, action = defaultAction) => {
         ...state,
         cron: action.payload.cron
       };
+    case ACTIONS.CRON_RESET:
+      return {
+        ...state,
+        intervalOption: INTERVAL_OPTIONS.DAILY,
+        minInterval: 5,
+        hourInterval: HOUR_OPTIONS_CLOCK[0],
+        dayInterval: DATE_OF_MONTH_OPTIONS[0],
+        daysOfWeekInterval: [1],
+        dateOfMonthInterval: DATE_OF_MONTH_OPTIONS[0],
+        monthInterval: MONTH_OPTIONS[0],
+        startingAtMinute: MINUTE_OPTIONS[0],
+        startingAtHour: HOUR_OPTIONS[0],
+        startingAtAMPM: AM_PM_OPTIONS[0]
```

Comment:
nit: Maybe have all these as `const DEFAULT_CRON_OPTIONS`, and have `DEFAULT_SCHEDULE_OPTIONS` extend it, so you don't have to repeat these values.